hub.yml

```diff
@@ -1,16 +1,16 @@
-name: policy-name
-description: DESCRIPTION OF YOUR POLICY
-homepage: POLICY HOMEPAGE URL
+name: psp-volumes
+description: PSP that controls usage of volumes
```

Comment:
nitpicking: replace PSP with `Pod Security Policy`, like we do on the other policies

Reply:
Woops, better, yes. Done in https://github.com/kubewarden/psp-volumes-policy/pull/1/commits/472c59059969cf5688bdf811c7634cbe2e353a41.